### ISSUE TITLE
Markdown preview links does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51856]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
ColorContrast,MAS1.4.3
**Repro Steps:**
1. Launch VS Code.
2. Navigate to Help(Alt+H) menu option and select "Interactive Editor Playground" from dropdown.
3.Open color contrast analyzer and check the ratio.

**Actual:**
Color Contrast ratio is less than 4.5:1 for the links
Same issue observed on other themes also.
**Expected:**
Color Contrast ratio should be greater than 4.5:1.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The user will not be able to see the controls and contents.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:** 
![accessibility a11y_vscode_interactiveeditorplayground_colorcontrast](https://user-images.githubusercontent.com/39204909/41397963-72ad6b9c-6fd3-11e8-8a1d-b6a6b96060a9.png)

**Comment by misolori** [comment]
The default link color meets the color contrast ratio but fails on the `Tip` background color:
![image](https://user-images.githubusercontent.com/35271042/43406280-bc02c780-93d0-11e8-9201-51e19bb01392.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
The issue is still reproduced on other themes.I verified on below environment
Version: 1.27.0-insider
Commit: 3f8088b5c5f1d44b5e1af4a8e67448abbe66470e
Date: 2018-08-27T07:36:47.732Z
Electron: 2.0.7
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

**Comment by misolori** [comment]
This is fixed in our default dark and light themes. Per our agreement, we will not be addressing third-party themes. cc @auchenberg 

Dark
![image](https://user-images.githubusercontent.com/35271042/44668335-f7799700-a9d1-11e8-83a2-801b81b6632b.png)


Light
![image](https://user-images.githubusercontent.com/35271042/44668331-f3e61000-a9d1-11e8-8f15-f07996043505.png)
